Resources/Private/Templates/NodeTypes/NewsFilter.html

```diff
@@ -22,4 +25,17 @@
             </li>
         </f:for>
     </ul>
-</f:section>
\ No newline at end of file
+</f:section>
+
+
+<f:section name="categoryFilter">
+    <ul class="news-categoryfilter">
+        <li class="headline">Themen / Kategorien</li>
+        <li><neos:link.node node="{node}"><span class="news-categoryfilter__category">Alle</span></neos:link.node></li>
+        <f:for each="{categoryFilter}" as="filter">
+            <li>
+                <neos:link.node node="{node}" arguments="{category: '{filter.properties.uriPathSegment}'}"><span class="news-categoryfilter__category">{filter.properties.title}</span></neos:link.node>
```

Comment:
We shouldn't access .properties on a a node within fluid as it slows down rendering a lot